packages/manager/modules/pci/src/projects/project/instances/instances.html

```diff
@@ -48,9 +48,14 @@
             data-sortable="asc"
         >
             <a
+                data-ng-if="$row.status !== 'DELETING'"
```

Comment:
We could use a `isDeleting()` in the instance model if there is one

Reply:
@marie-j This is what you had in mind, right?

See: #3850